internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -744,15 +780,13 @@ func (r *ClusterDeploymentsReconciler) updateIfNeeded(ctx context.Context,
 
 	// Update ignition endpoint if needed
 	if clusterInstall.Spec.IgnitionEndpoint != nil {
```

Comment:
Similar to above, allow unsetting the ignition endpoint (if it's nil and the clusterInstall.Spec.IgnitionEndpoint isn't nil)

Reply:
Done